build.gradle.kts

```diff
@@ -325,10 +325,23 @@ for (apiVersion in apiVersions) {
             includeTestsMatching(packagePrefix + "payment.international.scheduled.payments" + suffixPattern + apiVersion)
             includeTestsMatching(packagePrefix + "payment.international.standing.orders" + suffixPattern + apiVersion)
             includeTestsMatching(packagePrefix + "payment.file.payments" + suffixPattern + apiVersion)
+            if (apiVersion == ("v3_1_10")) {
+                includeTestsMatching(packagePrefix + "payment.domestic.vrp" + suffixPattern + apiVersion)
+            }
```

Comment:
Is this necessary? I would assume for versions with vrp packages then nothing matches so nothing gets added

Does gradle error without the if guard?